File: app/src/main/java/com/babestudios/companieshouse/data/model/company/BranchCompanyDetails.java
package com.babestudios.companieshouse.data.model.company;

import com.google.gson.annotations.SerializedName;

public class BranchCompanyDetails {

	@SerializedName("business_activity")
	public String businessActivity;

	@SerializedName("parent_company_name")
	public String parentCompanyName;

	@SerializedName("parent_company_number")
	public String parentCompanyNumber;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/company/BranchCompanyDetails.kt
package com.babestudios.companyinfouk.data.model.company

import com.google.gson.annotations.SerializedName

class BranchCompanyDetails {

	@SerializedName("business_activity")
	var businessActivity: String? = null

	@SerializedName("parent_company_name")
	var parentCompanyName: String? = null

	@SerializedName("parent_company_number")
	var parentCompanyNumber: String? = null
}

